src/pages/signin/SignInPage.tsx

```diff
@@ -299,6 +302,14 @@ function SignInPage({shouldEnableMaxHeight = true}: SignInPageInnerProps, ref: F
     useImperativeHandle(ref, () => ({
         navigateBack,
     }));
+
+    useEffect(() => {
+        if (!CONFIG.IS_HYBRID_APP || !hybridApp?.loggedOutFromOldDot) {
+            return;
+        }
+        HybridAppModule.clearOldDotAfterSignOut();
+    }, [hybridApp?.loggedOutFromOldDot]);
```

Comment:
Do we really have to do this in `useEffect`? 🤔 
If we decide to change `loggedOutFromOldDot` in some other place, this logic will fire :/

How about putting this in `Session/index.ts` in `setupNewDotAfterTransitionFromOldDot`?
cc @mateuuszzzzz for thoughts